src/context/Video/Watchlater.jsx

```diff
@@ -10,15 +11,15 @@ export const getWatchlater = async (videoDispatch) => {
     });
     const { status, data } = response;
     if (status === 200) {
-      videoDispatch({ type: "GET_WATCHLATER", payload: data?.watchlater });
+      userDispatch({ type: "GET_WATCHLATER", payload: data?.watchlater });
     }
   } catch (error) {
     Toast({ message: "Couldn't fetch Watchlist", type: "error" });
     console.log(error);
   }
 };
 
-export const addToWatchlater = async (video, videoDispatch) => {
+export const addToWatchlater = async (video, userDispatch) => {
   try {
     const response = await axios.post(
```

Comment:
you can do here 
const { status, data } = await axios.post(...);

so you can no need 34 line